mne/viz/misc.py

```diff
@@ -367,20 +368,32 @@ def plot_bem(subject=None, subjects_dir=None, orientation='coronal',
     if not op.isdir(bem_path):
         raise IOError('Subject bem directory "%s" does not exist' % bem_path)
 
-    surf_fnames = []
+    surfaces = {}
     for surf_name in ['*inner_skull', '*outer_skull', '*outer_skin']:
         surf_fname = glob(op.join(bem_path, surf_name + '.surf'))
         if len(surf_fname) > 0:
             surf_fname = surf_fname[0]
             logger.info("Using surface: %s" % surf_fname)
-            surf_fnames.append(surf_fname)
-
-    if len(surf_fnames) == 0:
+            surfaces[surf_fname] = 'yellow'
+
+    if brain_surfaces is not None:
+        if isinstance(brain_surfaces, string_types):
+            brain_surfaces = (brain_surfaces,)
+        for surf_name in brain_surfaces:
+            for hemi in ('lh', 'rh'):
+                surf_fname = op.join(subjects_dir, subject, 'surf',
+                                     hemi + '.' + surf_name)
+                if op.exists(surf_fname):
```

Comment:
My intuition would be to use try/except here ... doesn't require evaluating this expression, and is less words.